src/cfnlint/schemaManager.py

```diff
@@ -0,0 +1,103 @@
+"""
```

Comment:
I feel like this is a new naming standard?  The file name should be `schema_manager`